### ISSUE TITLE
UGRC data privacy and records management

### ISSUE THREAD

**Issue by nathankota** [agrc/planning-queue#251]
### Benefit

Compliance with state laws related to data privacy policies and practices. 

### Acceptance Criteria

- [ ] Assign/define a UGRC Administrative Records Officer (ARO) to work with DTS Chief Administrative Officer (CAO - Stephanie Weteling) to ensure the _care, maintenance, scheduling, disposal, classification, designation, access and preservation of records_, per Utah Code 63A-12-103(2).
- [ ] Work with CAO to create new and/or apply existing record series and retention schedules to UGRC records.
- [ ] Compose legal notification(s) and include in UGRC websites/pages that collect personal information, explaining the intended use and consequences of the data and the record series in which the data are included. 
    - **Question:** Do we also need a general notification for all SGID, etc. data? 

### Issue Reference

refs #

**Comment by steveoh** [comment]
@gregbunce silently became our certified records officer. I'm going to check that off.

**Comment by gregbunce** [comment]
happy to assit

**Comment by nathankota** [comment]
### Sprint 2 
- [ ] Attempt a notification statement for a site where we collect data, using [UIC](https://github.com/uic-utah/uic-inventory/issues/270) as a reference.

**Comment by steveoh** [comment]
Where are we at with this currently?

**Comment by gregbunce** [comment]
I haven't heard anything.

**Comment by steveoh** [comment]
I think this is a proactive task as opposed to someone telling us to do something if I remember correctly.

**Comment by brigsz** [comment]
After the finance meeting today at 12:30 I will look closer

**Comment by brigsz** [comment]
Okay, yes, proactive, however has Chris Bramwell put out an example of what sites should say etc?  I would expect some type of template to come from his office. FWIW I have not listened to the recording or read the transcript of the Chris Bramwell meeting from last year.  I would assume that all of DTS or the state for that matter would have the same pattern etc

**Comment by steveoh** [comment]
Is that what this bullet item intends to find out?

> Work with CAO to create new and/or apply existing record series and retention schedules to UGRC records.

**Comment by nathankota** [comment]
@brigsz can the second bullet be checked off, based on your response to the emails with Kate S. and her questions about prepping to send data to Archives, which made it sound like we have record series and retention schedules defined for "UGRC data"?

**Comment by brigsz** [comment]
[UGRCRecordSeries.pdf](https://github.com/user-attachments/files/15626690/UGRCRecordSeries.pdf)

For me, the second bullet does not make sense, I am not sure what is intended.  However I have attached our current record series list that may help clarify things

**Comment by nathankota** [comment]
> [UGRCRecordSeries.pdf](https://github.com/user-attachments/files/15626690/UGRCRecordSeries.pdf)
> 
> For me, the second bullet does not make sense, I am not sure what is intended. However I have attached our current record series list that may help clarify things

I also don't fully understand the intent, but the pdf you provided _seems_ to be related.

**Comment by brigsz** [comment]
Okay, then I feel the issue should read "Review and update the current record series and retention schedules for UGRC data"

On a side note, much of this came out of an effort called GEOMAPP with the Library of Congress, North Carolina and Kentucky.  Geospatial Multi-state Archive And Preservation Project (GEOMAPP)  The UGRC staff person just went super crazy on this and I believe a significant portion is overkill.  We should document the data we curate and use the appropriate schedules and classifications.  Other agencies should do the same for their data.  Where possible I think data should be thought of under the state geographic region as opposed to counties, so you have state wide parcels, state wide roads, state wide address points as opposed to calling out a record series for each county.  

Overall, yes, this is proactive and hopefully one day DTS and Archives will have a true plan.  

There is a formal way to submit UGRC data for archives, and I think Greg knows it.  There is a form you fill out and put the data out on a Google drive share where Archives can access it.

**Comment by gregbunce** [comment]
Thanks for backstory, Matt. That's helpful for me. 
On a side note, I also backup the SGID vector data every 6 months on the Google Drive.

**Comment by steveoh** [comment]
My memory was that we were going to shrink this PDF list to something more generic and manageable. A lot of those series listed don't really make sense to continue with e.g. Chukar records. I thought the intent of that bullet was to shrink it down to one item that would include all SGID data and the retention yada yada would reflect our current policies with porter.

**Comment by steveoh** [comment]
@brigsz would you track progress/updates on the records management here?

**Comment by brigsz** [comment]
@steveoh Yep, I will put the goods in here.  Just moving to this task within the next hour or so.

**Comment by brigsz** [comment]
When I look at the AGOL Shelved area, I see layers that have a date in the title.  It seems to me that those are the only ones in "Shelved" that I should be working with.  If UGRC does not curate, or is not a partner in the data then I don't think it should get a record series or retention schedules that falls under the purview of UGRC

**Comment by brigsz** [comment]
I am tempted to say the record series should be titled SGID Historic

**Comment by brigsz** [comment]
Okay, I need a boost to the next level.  So we have a layer, such as Tax Entities 2024 and we archive it in 2024. Now eventually it goes to shelved in AGOL and rests.  Our current retention schedule does not really contain the info that it will be in AGOL for a long long time.  however it has already been archived and the only thing incorrect is that its current retention schedules is wrong as we do something different in addition.  Is that what folks think?

**Comment by steveoh** [comment]
@leault We're currently absent a UGRC Administrative Records Officer which requires some training and yada yada. I think this issue has changed shape from its original intent, defining what we do with the data we collect within applications (Chris Bramwells - privacy officer) into the archive stuff which I believe should be tracked in a new feature.

Watching the recording should bring you up to speed on where Chris wants us to be.

**Comment by stdavis** [comment]
Should this be moved to a current/future quarter?

**Comment by steveoh** [comment]
I have a note to bring it up during the next lead meeting. It sounds like some things are in motion but no one is reporting on it. Does that seem right @eneemann?

**Comment by eneemann** [comment]
Laura and I were scheduled for "certification" training, but then it was determined that we didn't need to be certified.  Now we have an online training to review for a basic overview of records management while new training tailored towards DGO is developed.  That's the recent movement I'm aware of.
